faceoff.py

```diff
@@ -29,20 +66,21 @@ class FaceOffOutput(BaseInvocationOutput):
     y:                 int = OutputField(description="The y coordinate of the bounding box's top side")
 
 
-@invocation("face_off", title="FaceOff", tags=["image", "faceoff", "face", "mask"], category="image")
+@invocation("face_off", title="FaceOff", tags=["image", "faceoff", "face", "mask"], category="image", version="1.0.0")
 class FaceOffInvocation(BaseInvocation):
     """bound, extract, and mask a face from an image using MediaPipe detection"""
 
     image:               ImageField  = InputField(description="Image for face detection")
     face_id:             int = InputField(default=0, description="0 for first detected face, single digit for one specific. Multiple faces not supported. Find a face's ID with FaceIdentifier node.")
-    faces:               int = InputField(default=4, description="Maximum number of faces to detect")
+    faces:               int = InputField(default=99, description="Maximum number of faces to detect")
```

Comment:
Maybe we can just use 10 as a default. It's probably not likely anyone will need more than that and be caught unawares.
```suggestion
    faces:               int = InputField(default=10, description="Maximum number of faces to detect")
```